components/layouts/columns/columns.hbs

```diff
@@ -2,10 +2,25 @@
   <div class="columns section__inner">
 
     <h3>Dedicated consultants</h3>
-    <p>Our Direct Hire Consultants work only on permanent hiring and are dedicated to a limited number of positions at once so they can give you the time and attention you need.</p>
+    <p class="themed-text">Our Direct Hire Consultants work only on permanent hiring and are dedicated to a limited number of positions at once so they can give you the time and attention you need.</p>
     
     <h3>Top-tier talent</h3>
-    <p>Direct Hire Consultants draw from the quent global talent network to help you reach talent you couldn’t access yourself. This includes those superstars who aren’t looking but are open to the right opportunity.</p>
+    <p class="themed-text">Direct Hire Consultants draw from the quent global talent network to help you reach talent you couldn’t access yourself. This includes those superstars who aren’t looking but are open to the right opportunity.</p>
+
+    <h3>Collaborative partnerships</h3>
+    <p>Consultants deeply understand recruiting in marketing, creative, and experience design, adding value at every step. Collaborating closely with you, they’ll guide you through the process of finding your next great hire.</p>
```

Comment:
Do we need a themed-text class here?

Reply:
I left that without the class for testing purposes, like to see the difference. I'm not sure if it's necessary. What do you think?

Reply:
Makes sense. Not a blocker.